doc/parse_gx_xsd.py

```diff
@@ -0,0 +1,201 @@
+# coding: utf-8
+# TODO: Add examples, tables and best practice links to command
+# TODO: Examples of truevalue, falsevalue
+# TODO: Test param extra_file
+from __future__ import print_function
+
+# Things dropped from TOC (still documented inside schema).
+#  - request_parameter_translation
+
+from lxml import etree
+from StringIO import StringIO
+
+
+with open("doc/schema_template.md", "r") as f:
+    MARKDOWN_TEMPLATE = f.read()
+
+with open("lib/galaxy/tools/xsd/galaxy.xsd", "r") as f:
+    xmlschema_doc = etree.parse(f)
+
+markdown_buffer = StringIO()
+
+
+def main():
+    """Entry point for the function that builds Markdown help for the Galaxy XSD."""
+    for line in MARKDOWN_TEMPLATE.splitlines():
+        if line.startswith("$tag:"):
+            print(Tag(line).build_help())
+        elif line.startswith("$toc"):
```

Comment:
I wanted to highlight that `$toc` should be the only thing in the line, not a prefix. I.e. break the consistency :stuck_out_tongue_winking_eye: